src/components/Claim/Claim.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import Claim from './Claim';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Components/Claim',
+  component: Claim,
+} as ComponentMeta<typeof Claim>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof Claim> = (args) => <Claim {...args} />;
+
+export const Default = Template.bind({});
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+Default.args = {
+  claim: {
+    claim:
+      'Viral video of the moon rising in the North Pole is computer generated',
+    domain: 'https://www.reuters.com/favicon.ico',
```

Comment:
Should be `reuters.com`, otherwise the favicon isn't displayed correctly. `https://` and `/favicon.ico` gets added in Claim.tsx already so this doubles it.

Reply:
This one and down below are really good catches!

Reply:
Fixed both